lib/rules/require-capitalized-comments.js

```diff
@@ -52,7 +52,7 @@
  * ```
  *
  * ```js
- * "requireCapitalizedComments": { allExcept: ["jshint"] }
+ * "requireCapitalizedComments": { "allExcept": ["jshint"] }
```

Comment:
No worries, it took me a while to spot it myself just to realize it was a
bad copy paste from the docs.
On Apr 29, 2015 10:07 PM, "Eli White" <notifications@github.com> wrote:

> In lib/rules/require-capitalized-comments.js
> <https://github.com/jscs-dev/node-jscs/pull/1322#discussion_r29369324>:
>
> > @@ -52,7 +52,7 @@
> >   * ```
> >   *
> >   * ```js
> > - * "requireCapitalizedComments": { allExcept: ["jshint"] }
> > + * "requireCapitalizedComments": { "allExcept": ["jshint"] }
>
> Ah yes, json, not js. Ignore me. ;)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jscs-dev/node-jscs/pull/1322/files#r29369324>.
>